go.mod

```diff
@@ -73,4 +73,4 @@ require (
 	honnef.co/go/tools v0.1.4
 )
 
-replace github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
```

Comment:
The protobuf dependency has been removed, as the one used in the main section is the same.